site/content/docs/5.1/forms/quantity-selector.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: docs
+title: Quantity selector
+description: Form element used to select a number.
+group: forms
+toc: true
+---
+
+## Examples
+
+Quantity selector is a form element used to select a number. The default version is the small version. To use the large version, use the contextual class `.quantity-selector-l`.
```

Comment:
I don't understand why `The default version is the small version.`. The default version should be the one with a height of 40px to be used in a normal form (with other inputs, buttons, etc.)

Reply:
Done